.github/workflows/shopify-cli-extensions-testing.yml

```diff
@@ -1,21 +1,29 @@
-name: Testing
+name: shopify-cli-extensions-testing
```

Comment:
Let's use the same naming convention throughout - currently the extentions ones are all sentences